api/django-fileupload/django_fileupload/views.py

```diff
@@ -100,4 +101,15 @@ class FileUploadViewSet(
     mixins.DestroyModelMixin,
     viewsets.GenericViewSet,
 ):
+    keep_after_deletion = False
+
+    def destroy(self, request, *args, **kwargs):
+        file_upload = self.get_object()
+        if self.keep_after_deletion:
+            file_upload.deleted_on = timezone.now()
```

Comment:
So deleted files should not appear in the file upload list right? Where are they filtered out?

Reply:
It could happen that you have a case in which you want to see also deleted files (e.g. some kind of admin panel with very specific rights) that's why I left this type of filtering to the user in their app.